<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1314</width>
    <height>774</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="chartWykres2" native="true">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>340</y>
      <width>481</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="chartwykres3" native="true">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>30</y>
      <width>481</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="chartWykres4" native="true">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>330</y>
      <width>481</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="chartWykres1" native="true">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>481</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="STATUS">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>271</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>STATUS</string>
    </property>
    <widget class="QLabel" name="lblCzas">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>271</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>PARAMETRY SYMULACJI:</string>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Interwal:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinInterwal">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>200</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>271</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>GENERATOR WARTOSCI ZADANEJ</string>
    </property>
    <widget class="QComboBox" name="comboTyp">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>72</width>
       <height>24</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Prostokąt</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sinusoida</string>
      </property>
     </item>
    </widget>
    <widget class="QSpinBox" name="spinAmp">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinOkres">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinWypelnienie">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>120</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>TYP:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Amplituda:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Okres:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Wypłnienie:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>271</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>PARAMETRY PID</string>
    </property>
    <widget class="QDoubleSpinBox" name="spinKp">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>62</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinTi">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>62</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinTd">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>90</y>
       <width>62</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Kp:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ti:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Td:</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnWczytaj">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>660</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>WCZYTAJ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnZapisz">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>660</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>ZAPISZ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReset">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>660</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>RESET</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStop">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>660</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>660</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUstawieniaARX">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>660</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>USTAWIENIA ARX</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnResetCalki">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>660</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>RESET CAŁKI</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1314</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
